libavfilter/vf_livepeer_dnn.c

```diff
@@ -204,17 +175,17 @@ static int filter_frame(AVFilterLink *inlink, AVFrame *in)
     AVFilterLink *outlink = context->outputs[0];
     DNNReturnType dnn_result;
 
-    AVFrame *out = ff_get_video_buffer(outlink, outlink->w, outlink->h);
-    if (!out){
+    AVFrame *out = ff_get_video_buffer(outlink, ctx->output_width, ctx->output_height);
```

Comment:
`ctx->output_width` and `ctx->output_height` are set to the dimensions of the Model output (which would be `3x1` for the 3 probability values)

In my end-to-end tests with go-livepeer using this branch, this seemed to give errors:
```
I0910 11:22:16.046514   30549 lb.go:207] LB: Transcode submitted for b0416cd9_0 
[IMGUTILS @ 0x7f64b6d1f3a0] Picture size 0x3 is invalid
[livepeer_dnn @ 0x7f626afabc80] could not allocate memory for output frame 
```

Now that I think of it, we can do away with allocating the `out` frame altogether, and instead pass the `in` frame as the output to `ff_dnn_execute_model`.

I'll make that change and send an update within this PR's branch.

Reply:
After discussing with @cyberj0g decided to instead hardcode the output frame buffer to be of `64x64` width x height in https://github.com/livepeer/FFmpeg/pull/12/commits/838fceed93a8c1494dea2e53af62c804794dac83

It's better to pass-through the `in` frame untouched, even though we only update the metadata, as I am not 100% sure how ffmpeg filtergraph handles free-ing modified frames.